widget/embedded/src/containers/Inputs/Inputs.tsx

```diff
@@ -140,11 +146,11 @@ export function Inputs(props: PropTypes) {
           usdValue: priceImpactOutputCanNotBeComputed
             ? undefined
             : numberToString(
-                outputUsdValue,
+                outputUsdAmount,
                 USD_VALUE_MIN_DECIMALS,
                 USD_VALUE_MAX_DECIMALS
               ),
-          realValue: formatTooltipNumbers(outputAmount),
+          realValue: formatTooltipNumbers(outputUsdAmount),
```

Comment:
this is real output value, not real output USD value,
now, the tooltip for the real output value is incorrect,
Please revert this to its previous state.

Please make sure that all input and output items are correct for both the main value and its tooltip value and the dollar value and its tooltip value.

Reply:
fixed